source/content/quicksilver.md

```diff
@@ -58,6 +57,13 @@ You can hook into the following workflows:
 | `sync_code`                            | Push code via Git or commit OSD/SFTP changes via Pantheon Dashboard | Dev or Multidev            |                                             |
 | `create_cloud_development_environment` | Create Multidev environment                                         | Multidev                   | `after` stage valid, `before` stage invalid |
 
+## Secrets
+
+Your script may require tokens, passwords, or other information that should be protected. These values should be stored in securely. Some methods for secure storage include:
+
+- A key management service like [Lockr](/guides/lockr),
+- A storage solution in your site's [private files path](/private-paths#private-path-for-files),
+- A secrets file managed by the [Terminus Secrets Plugin](https://github.com/pantheon-systems/terminus-secrets-plugin).
```

Comment:
The second and third bullets are pretty much the same -- or, more to the point, the third bullet is just one example of a way to achieve the goals of the second bulltet.

Maybe remove the third bullet, and instead add a bit of text after the list:

> The Terminus Secrets Plugin provides a convenient way to manage secret data in json files in your site's private files path.